src/neuroconv/datainterfaces/ophys/baseimagingextractorinterface.py

```diff
@@ -77,11 +77,12 @@ def run_conversion(
         metadata: Optional[dict] = None,
         overwrite: bool = False,
         stub_test: bool = False,
+        stub_frames: int = 100,
         save_path: OptionalFilePathType = None,
     ):
 
         if stub_test:
-            stub_frames = min([100, self.imaging_extractor.get_num_frames()])
+            stub_frames = min([stub_frames, self.imaging_extractor.get_num_frames()])
```

Comment:
I needed this one for my current conversion since even 5 frames is almost half a GB of data. This kind of thing is really only needed for Imaging of volumetric data, ecephys interfaces should have no trouble similar to this for a very long time.

That said, it is perhaps tangential to the rest of the PR so I could split it into a separate PR if requested

Reply:
This is good to have here, I'm just thinking whether it would make sense to have `stub_options` as dict instead of just the `stub_frames`, but I guess it can be changed later if we wanted to stub by image size later?

Reply:
`stub_options` isn't a bad idea in the long run, as we could include the `stub_test: bool` functionality within it, such as `enable: bool`. Would also be useful for channel-wise stubbing in ecephys for the previous observation interval argument.

That's probably its own entire thing though... especially as we'd have to start a definition for the sub-schema of that argument. Maybe best left for subsequent PR?